embassy-rp/src/spi.rs

```diff
@@ -52,7 +54,7 @@ impl<'d, T: Instance> Spi<'d, T> {
 
             // Find largest post-divide which makes output <= baudrate. Post-divide is
             // an integer in the range 1 to 256 inclusive.
-            let postdiv = (1u32..=256)
+            let postdiv = (2u32..=256)
```

Comment:
Is this comment outdated now ?

Reply:
Maybe. There are crash issues when setting a too high baudrate, I still have to look at them. This diff should definitely not be here anyway, will remove.

Reply:
Left it TODO for now, the calculation with the loop is stupid and needs refactoring anyway.